jekyll/_cci2/configuration-reference.adoc

```diff
@@ -2628,9 +2628,46 @@ Jobs are run concurrently by default, so you must explicitly require any depende
 | `requires`
 | N
 | List
-| A list of jobs that must succeed for the job to start. Note: When jobs in the current workflow that are listed as dependencies are not executed (due to a filter function for example), their requirement as a dependency for other jobs will be ignored by the requires option. However, if all dependencies of a job are filtered, then that job will not be executed either.
+| A list of jobs that must succeed or attain a specified status for the job to start. Note: When jobs in the current workflow that are listed as dependencies are not executed (due to a filter function for example), their requirement as a dependency for other jobs will be ignored by the requires option. However, if all dependencies of a job are filtered, then that job will not be executed either.
 |===
 
+[.table.table-striped]
+[cols=3*, options="header", stripes=even]
+|===
+| Key | Type | Description
+
+| `job_name`
+| Job Name
+| A required job that must `succeed` for the job to start.
+
+| `job_name`
+| Map
+| A map of job name to a status or list of statuses.
+
+  The job starts if the required dependency reaches one of the statuses. The possible status values are: `success`, `failed` and `canceled`.  
+|===
+
+[,yml]
+----
+workflows:
+  my-workflow:
+    jobs:
+      - build
+      - test:
+          requires:
+            - build
+      - notify-canceled:
+          requires:
+            - build: canceled
+      - deploy:
+          requires:
+            - build
+            - test:
+              - success
+              - failed
```

Comment:
Does it make sense to have an example where you deploy on failed or passed tests, just not cancelled?

Reply:
I can rename the `deploy` job to be `share-test-results` for example. Would that make more sense?

Reply:
If we want a realistic example, deploying on failed isn't really a use case people typically want. A common use case for this feature is to run a clean up job when a deploy fails to tear down infrastructure/resources.

So we could have a `deploy` job that requires `build` and `test`. Then a `cleanup` job that requires `deploy` to fail.

For example:
```yaml
workflows:
  my-workflow:
    jobs:
      - build
      - test:
          requires:
            - build
      - deploy:
          requires:
            - build
            - test
     - notify-slack:
          requires:
            - build: canceled
      - cleanup:
          requires:
            - deploy:
              - failed
              - canceled
```